src/l10n/fr.json

```diff
@@ -0,0 +1,116 @@
+{
+    "translations": {
+        "Passwords"                                                                                 : "Mots de passe",
+        "HTTPS Required"                                                                            : "HTTPS Requis",
+        "reload this page"                                                                          : "Recharger cette page",
+        "documentation"                                                                             : "Documentation",
+        "This application requires %s in order to work safely."                                     : "Cette application dépend de %s afin de fonctionner correctement",
+        "Consult the %s to configure Nextcloud correctly."                                          : "Consultez %s afin de configurer Nextcloud correctement",
+        "If you do not have a HTTPS certificate, get one for free from %s."                         : "Si vous n’avez aucun certificat HTTPS vous pouvez en avoir un gratuitement un ici %s.",
+        "Enable %s to troubleshoot this error."                                                     : "Activez %s pour résoudre cette erreur.",
```

Comment:
The `%s` will be filled with the string "HTTPS debugging" which seems to be missing in the German files. Since the option in the admin settings is translated, it might be useful to add the translation for this piece as well.